src/cpp/src/lora/adapter.cpp

```diff
@@ -373,47 +433,76 @@ struct LoRAWeightStateGetter {
                 params->type,
                 variable_id_prefix + ".A"
             };
-            result.A = add_variable(var_ids.A);
+            result.A = add_variable(var_ids.A, model);
             // FIXME: No guarantees on ordering of state in InferRequest makes impossible using indices of variables later, forced to use variable_id instead
             //indices.A = model->get_variables().size();
             var_ids.alpha = ov::op::util::VariableInfo{
                 params->fine_grained_alpha ? ov::PartialShape{1, params->rank} : ov::PartialShape{},
                 ov::element::f32,   // alpha is always f32 because it is set from host as float data type
                 variable_id_prefix + ".alpha"
             };
-            result.alpha = add_variable(var_ids.alpha);
+            result.alpha = add_variable(var_ids.alpha, model);
             // FIXME: No guarantees on ordering of state in InferRequest makes impossible using indices of variables later, forced to use variable_id instead
             //indices.B = model->get_variables().size();
             var_ids.B = ov::op::util::VariableInfo{
                 ov::PartialShape{output_dim, params->rank},  // Will be used with transpose_b == true
                 params->type,
                 variable_id_prefix + ".B"
             };
-            result.B = add_variable(var_ids.B);
+            result.B = add_variable(var_ids.B, model);
             variable_ids.emplace(name, var_ids);
             return result;
         } else {
             return std::nullopt;
         }
     }
+};
 
-    NodePtr add_variable(const ov::op::util::VariableInfo& variable_info) const {
-        auto variable = std::make_shared<ov::op::util::Variable>(variable_info);
-        model->add_variables({variable});
-        #if 0
-        // Attempt to pre-build initialization expression with empty tensors that should discard LoRA effect by default
-        // FIXME: CPU plugin fails when there is no initialization expression is given and type is not fp32
-        ov::Shape init_shape(shape.rank().get_length());
-        for(size_t i = 0; i < shape.size(); ++i) {
-            init_shape[i] = shape[i].get_min_length();
+struct LoRAStateGetterForConst {
+    std::shared_ptr<ov::Model> model;
+    LoRAConstantGetter getter;
+    std::map<std::string, ov::op::util::VariableInfo>& variable_ids;
+    const std::string if_variable_id = "lora_state_0_replace_orig_constant";
```

Comment:
`if_variable_id` is only used in `create_if_input()`. Move `if_variable_id` into the function so readers don't need to jump around for it